single-node-refactor/src/Solvers/SGH_solver/include/sgh_solver.h

```diff
@@ -130,6 +134,23 @@ class SGH : public Solver
 
         std::cout << "Applying initial boundary conditions" << std::endl;
         boundary_velocity(mesh, sim_param.boundary_conditions, node.vel, time_value);
+
+        // Setting up contact
+        for (size_t i = 0; i < mesh.num_bdy_sets; i++) {
```

Comment:
Like in my previous comment, we should pull this out of the SGH solver.  For now, leave it in, that can be a future PR once we figure out where it needs to live.

Reply:
Got it